net-libs/loudmouth/loudmouth-1.5.3-r1.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Lightweight C Jabber library"
+HOMEPAGE="https://github.com/mcabber/loudmouth"
+SRC_URI="https://github.com/mcabber/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~ppc-macos"
+
+IUSE="asyncns ssl openssl static-libs test"
+
+# Automagic libidn dependency
+RDEPEND="
+	>=dev-libs/glib-2.16:2
+	net-dns/libidn
+	ssl? (
+		!openssl? ( >=net-libs/gnutls-1.4.0 )
+		openssl? ( dev-libs/openssl:0 )
```

Comment:
Maybe dev-libs/openssl:0= will be better

Reply:
Yep.  Done.